### DISCUSSION TITLE
Is not possible to rerun the same launcher from the module

### DISCUSSION THREAD

**Discussion by felipefacundes** [Alexays/Waybar#1816] [Q&A]
When executing a program by custom / module. It only runs once. Unfortunately, it is not possible to rerun the same launcher from the module.

**Comment by Alexays** [comment]
You had to set an interval in the custom module config

**Comment by felipefacundes** [comment]
Both rofi and dolphin releases this bug. This is my config

https://raw.githubusercontent.com/felipefacundes/dotfiles/master/waybar/config

**Comment by Alexays** [comment]
Juste replace `bash -c '$HOME/.config/rofi/rofimenu.sh &'` with `bash -c '$HOME/.config/rofi/rofimenu.sh'` or just `$HOME/.config/rofi/rofimenu.sh`

**Comment by felipefacundes** [comment]
I did that and the bug remains

**Comment by felipefacundes** [comment]
And this BUG also happens with dolphin, nautilus, nemo and thunar

"on-click": "dolphin",

Or

"on-click": "nautilus",

Opening once cannot reopen

**Comment by cafehaine** [comment]
those programs are probably similar to thunar when it never really closes after it was launched, in order to launch faster the next time.
From what I see, waybar cannot launch a program while the first instance is still running, so that might be your problem.
Dolphin or nautilus might have a way to start them as daemon in the background, which would solve this problem.

**Comment by felipefacundes** [comment]
I already did it for daemon. Even so this problem persists. This problem does not happen with any other panels. Not with polybar. Not even with lemonbar. Not even with tint2. No other. So this is a BUG on the waybar.
